### PR TITLE
Dockerfile: include git

### LINKED ISSUE #203: Git in docker container.
I believe Git just be installed by default in the docker container. Git integration is a core feature of VSCode. It's easy to add `apt-get install git` without significant complication or inflation of the docker image.

### PR DESCRIPTION
**Pull request by nhooyr**
Also improves installation of runtime deps.

Closes #221
Closes #230
Closes #203

### CODE REVIEW

**Comment by ammario** [comment]
I think we should merge in everyone else's PRs instead so they get attribution.

**Comment by pokemonlover1234** [comment]
Mine (#230) does the same thing, except I put the git install in the wrong spot.

**Comment by nhooyr** [comment]
@ammario the changes in this PR are different from the other PRs. Regardless, I've included all their commits here so they still get attribution and then included my own commit at the end to fix things up.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM node:8.15.0
 
-# Install VS Code's deps. These are the only two it seems we need.
+# Install VS Code's deps. These are the only two it seems we need. Install git for source control.
```
Comment (ammario) [inline Dockerfile]: Second sentence is in the wrong place (and extraneous imo)

Comment (nhooyr) [reply Dockerfile]: I agree, from old a commit I just added. Will remove.

File: Dockerfile [hunk 2]
```diff
@@ -20,11 +20,13 @@ FROM ubuntu:18.10
...
+
 RUN apt-get update && apt-get install -y \
 	openssl \
-	net-tools
-RUN apt-get install -y locales && \
-	locale-gen en_US.UTF-8
+	net-tools \
+	git \
+	locales
+RUN locale-gen en_US.UTF-8
```
Comment (SuperSandro2000) [inline Dockerfile]: We could squash that into the other run.

**Pull request merged**